Sources/SwiftSyntaxMacroExpansion/MacroExpansion.swift

```diff
@@ -12,6 +12,7 @@
 
 import SwiftSyntax
 @_spi(MacroExpansion) import SwiftSyntaxMacros
+import SwiftSyntaxBuilder  // FIXME: Why is HasTrailingOptionalCodeBlock there?
```

Comment:
Would you mind defining a trait for `HasTrailingCodeBlock` and  `HasTrailingOptionalCodeBlock`?

Reply:
Turns out that we have `WithCodeBlock`, so I'll use that and add `WithOptionalCodeBlock`

Reply:
https://github.com/apple/swift-syntax/pull/2359